src/stores/ui.store.tsx

```diff
@@ -965,17 +964,24 @@ export let createUIStore = (root: IRootStore) => {
       // })
 
       solNative.getApps().then(apps => {
+        let systemPreferencesUrl: string | undefined = undefined
+
         // Each "app" is a macOS file url, e.g. file:///Applications/SF%20Symbols
         const cleanApps = apps.map(url => {
           const pureUrl = decodeURI(url.replace('file://', ''))
           const tokens = pureUrl.split('/')
+          const name = tokens[tokens.length - 2].replace('.app', '')
+          // TODO: find a more i18n friendly way to find the system preferences app
+          if (name === 'System Preferences') systemPreferencesUrl = pureUrl
```

Comment:
@ospfranco Do you have any suggestions for this todo?

Reply:
not really sure what is this supposed to do?